### PR TITLE
DDF-4050 Fix Back button font

### PR DESCRIPTION
**Pull request by mojogitoverhere**
#### What does this PR do?
Fixes font of the Back button when paging through results
#### Who is reviewing it? 

@GabrielFabian 	@gjvera 	@djblue 	@willwill96 	
#### Select relevant component teams: 
@codice/ui 

#### Ask 2 committers to review/merge the PR and tag them here.

@andrewkfiedler
@bdeining
#### How should this be tested?

1. Ingest n documents
2. Set the `Number of Search Results` to a number less than n (settings -> Search Options)
3. Verify the Back button looks good
#### What are the relevant tickets?
[DDF-4050](https://codice.atlassian.net/browse/DDF-4050)
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: ui/packages/catalog-ui-search/src/main/webapp/component/paging/paging.hbs [hunk 1]
```diff
@@ -37,7 +37,7 @@
 {{#if showNextServerPage}}
     <button class="server-next is-primary">
         <span>More</span>
-        <span class="fa fa-angle-double-right"/>
+        <span class="fa fa-angle-double-right"></span>
```
Comment (mojogitoverhere) [inline ui/packages/catalog-ui-search/src/main/webapp/component/paging/paging.hbs]: I also updated the "More" button span for consistency. It still looks the same.

File: ui/packages/catalog-ui-search/src/main/webapp/component/paging/paging.hbs [hunk 2]
```diff
@@ -20,7 +20,7 @@
 <div class="buttons-left">
 {{#if showPreviousServerPage}}
     <button class="server-previous is-primary">
-        <span class="fa fa-angle-double-left"/>
+        <span class="fa fa-angle-double-left"></span>
```
Comment (mojogitoverhere) [inline ui/packages/catalog-ui-search/src/main/webapp/component/paging/paging.hbs]: The self-closing tag was causing the "Back" span to be nested within this span for some reason, causing the `fa` class to apply a different `font-family` to the "Back" text. Explicitly closing the span prevents this.

**Comment by mojogitoverhere** [comment]
build now

**Comment by GabrielFabian** [comment]
hero in progress

**Comment by GabrielFabian** [comment]
🎉 hero build successful 🎉 
screen shot 2018-10-23 at 12 50 18 pm

**Pull request merged**